Exec/RegTests/Spray-Jet/example.inp

```diff
@@ -0,0 +1,127 @@
+pelec.sum_interval   = -1   # timesteps between computing mass
+pelec.v              = 0   # verbosity in Castro.cpp
+amr.v                = 1    # verbosity in Amr.cpp
+#amr.data_log         = datlog
+#amr.grid_log         = grdlog  # name of grid logging file
+
+# REFINEMENT / REGRIDDING
+amr.max_level       = 2       # maximum level number allowed
+
+# Use fixed refinement base on an initial grid file
+#amr.initial_grid_file = gridfile
+#amr.regrid_int      = -1 -1 -1 # how often to regrid
+
+# Use adaptive refinement based on temperature gradients
+tagging.tempgrad = 1.3
+# Use adaptive refinement based on velocity gradients
+#tagging.velgrad = 40.
+amr.regrid_int = 4 8 8
+
+amr.ref_ratio       = 2 2 2 2 # refinement ratio
+amr.blocking_factor = 16       # block factor in grid generation
+amr.max_grid_size   = 32
+amr.n_error_buf     = 2 2 2 2 # number of buffer cells in error est
+
+# PARTICLES / SPRAY
+pelec.do_spray_particles = 1
+particles.v = 0
+particles.derive_plot_vars = 1
+particles.mom_transfer = 1
+particles.mass_transfer = 1
+particles.write_ascii_files = 0
+
+particles.fuel_ref_temp = 298.15
+
+particles.fuel_species = NC10H22
+# properties for decane
+particles.NC10H22_crit_temp = 617.8 # K
+particles.NC10H22_boil_temp = 447.27 # K
+particles.NC10H22_latent = 3.5899E9
+particles.NC10H22_cp = 2.1921E7 # Cp at 298 K
+particles.NC10H22_rho = 0.640
+
+# Coefficients for saturation pressure using Antoine equation
+# These are from the NIST website
+# Last coefficient converts units, in this case bar, to dyne/cm^2
+particles.NC10H22_psat = 4.07857 1501.268 -78.67 1.E6
+
+# particles.wall_temp = 430.
+particles.use_splash_model = false
+
+# CHECKPOINT FILES
+amr.checkpoint_files_output = 0
+amr.check_file      = chk  # root name of checkpoint file
+amr.check_int       = -1   # number of timesteps between checkpoints
+
+# PLOTFILES
+#amr.plot_files_output = 0
+amr.plot_file       = plt # root name of plotfile
+amr.plot_per        = 1.E-4 # number of timesteps between plotfiles
+amr.plot_vars = density Temp rho_E xmom ymom zmom rho_O2 rho_N2 rho_NC10H22
+amr.derive_plot_vars = x_velocity y_velocity z_velocity pressure   # these variables appear in the plotfile
+amr.file_name_digits = 6
+
+# PROBLEM
+# Initial fluid properties
+prob.ref_T = 440.
+prob.ref_p = 6.E7
+prob.init_v = 0.
+prob.init_N2 = 1.
+prob.init_O2 = 0.
+
+# Properties of injected particles
+spray.jet1.T = 363.
+spray.jet1.dist_type = Uniform
+spray.jet1.diameter = 4.5E-4
+spray.jet1.jet_dia = 9.E-3
+spray.jet1.spread_angle = 21.
+spray.jet1.jet_cent = 0. 0. 0.
+spray.jet1.jet_norm = 0. 1. 0.
+
+# These are necessary if an injection reference file is not set
+# spray.jet1.jet_vel = 6.E4
+# spray.jet1.mass_flow_rate = 2.3
+
+# ROI file can be generated at
+# https://www.cmt.upv.es/#/ecn/download/InjectionRateGenerator/InjectionRateGenerator
+spray.jet1.roi_file = roi.dat
+# Copied from Spray-A-Wbreakup
+spray.jet1.discharge_coeff = 0.89
+
+# UNUSED PARMPARSE
```

Comment:
@ldowen why are these not used? deprecated or renamed or something else?

Reply:
The `discharge_coeff` used to be called `cd` and the mass fraction is only read in if the case is compiled with multiple liquid species.